src/test/java/com/es/phoneshop/web/ProductListPageServletTest.java

```diff
@@ -24,18 +28,26 @@
     private HttpServletResponse response;
     @Mock
     private RequestDispatcher requestDispatcher;
+    @Mock
+    private ProductDao productDao;
 
-    private ProductListPageServlet servlet = new ProductListPageServlet();
+    @InjectMocks
+    private ProductListPageServlet servlet ;
 
     @Before
     public void setup(){
+        MockitoAnnotations.initMocks(this);
```

Comment:
Is this line actually required to initialize fields when using MockitoJUnitRunner? Double check that please

Reply:
I've tested it locally and it's not, so please remove it